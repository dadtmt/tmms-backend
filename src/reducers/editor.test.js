import reducer from './editor'

describe('createPage mutation result', () => {
  it('set page id', () => {
    const action = {
      operationName: 'CreatePage',
      result: {
        data: {
          createPage: {
            changedPage: {
              id: 'SOME_ID'
            }
          }
        }
      },
      type: 'APOLLO_MUTATION_RESULT'
    }
    expect(
      reducer({ currentPageId: null }, action)
    ).toMatchSnapshot()
  })
})

describe('editor reducer', () => {
  it('return initialState on some action', () => {
    expect(
      reducer({ currentPageId: null }, { type: 'SOME_ACTION' })
    ).toMatchSnapshot()
  })
})
