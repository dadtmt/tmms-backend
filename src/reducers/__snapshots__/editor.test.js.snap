exports[`createPage mutation result set page id 1`] = `
Object {
  "currentPageId": "SOME_ID",
}
`;

exports[`editor reducer return initialState on some action 1`] = `
Object {
  "currentPageId": null,
}
`;
